deploy/infra/aws/aws-native-infra/terraform/infra/input.tf

```diff
@@ -75,6 +75,26 @@ variable "eks_additional_role_mapping" {
   description = "List of additional roles mapping for aws_auth map. With this parameter you can configure access to the EKS cluster for AWS IAM entities."
 }
 
+variable "eks_cloudwatch_logs_retention_in_days" {
+  type        = number
+  default     = 30
+  description = "Possible values are: 1, 3, 5, 7, 14, 30, 60, 90, 120, 150, 180, 365, 400, 545, 731, 1096, 1827, 2192, 2557, 2922, 3288, 3653, and 0. If you select 0, the events in the log group are always retained and never expire."
+}
+
+###################################################################
+#                  Optional file systems
+###################################################################
+
+variable "deploy_filesystem_type" {
+  type        = string
+  default     = "efs"
+  description = "Option to create EFS or FSx Lustre filesystem: must be set efs or fsx.If leave as is, neather will be created."
+  validation {
+    condition     = contains(["efs", "fsx"], var.deploy_filesystem_type)
+    error_message = "The value of the deploy_filesystem_type variable can be only efs or fsx. Please check if variable set correctly"
```

Comment:
check **that** variable **is** set correctly

Reply:
Fixed